src/LaraCart.php

```diff
@@ -465,7 +466,7 @@ public function taxTotal($format = true)
      *
      * @return string
      */
-    public function total($format = true, $withDiscount = true)
+    public function total($format = true, $withDiscount = true, $withTax = true)
```

Comment:
total should always contain tax . if you need it without tax you can use subtotal